packages/common/duration.ts

```diff
@@ -0,0 +1,19 @@
+const millisecond = 1;
```

Comment:
we already have this defined here ts/packages/model/duration.ts

Reply:
can you pull it in by adding `@formant/model` as a dependency? or maybe we don't publish that... if not let's at least put this under ts/packages/model ...